apps/play/src/components/SampleListSection.tsx

```
@@ -0,0 +1,83 @@
+
+  return (
+    <div>
+      {loading() ? (
+        <div>Loading...</div>
+      ) : (
+        <For each={samples()}>
+          {(sample) => (
+            <div
+              class='sample-item'
+              onclick={() => props.onSampleSelect(sample)}
+            >
```

Biome (2.5.0)

[error] Unexpected event handler on static element.

(lint/a11y/noStaticElementInteractions)